build_artifacts.jl

```diff
@@ -2,25 +2,34 @@
 # it into `Artifacts.toml`.
 # https://github.com/JuliaImages/TestImages.jl/issues/67
 #
-# Github action "Upload Artifacts" runs this script and uploads the generated artifact file
+# GitHub action "Upload Artifacts" runs this script and uploads the generated artifact file
 
 using SHA
+using TOML
 import Tar
 import TranscodingStreams: TranscodingStream
 import CodecZlib: GzipCompressor, GzipDecompressor
 
 compress(io::IO) = TranscodingStream(GzipCompressor(level=9), io)
 decompress(io::IO) = TranscodingStream(GzipDecompressor(), io)
 
-
+# Generate images.tar.gz
 tarball, tree_path = abspath.(("images.tar.gz", "images"))
 open(tarball, write=true) do io
     close(Tar.create(tree_path, compress(io)))
 end
 
+# Check the filenames in images/metadata.toml
+metadata = TOML.parsefile(joinpath("images","metadata.toml"))
+filenames_meta = [img["name"] for img in metadata["images"]]
+filenames_dir = [splitext(name)[1] for name in readdir("images")]
+@assert filenames_meta ⊆ filenames_dir
```

Comment:
It looks like this should also be made as a part of CI during the PR. Because currently when someone propose a new image to `images` branch, there is no running CI.

Of course, since the changes are usually simple, a manual check still works.

Reply:
>It looks like this should also be made as a part of CI during the PR.

I agree with that. I can work on it when I have time.